### DISCUSSION TITLE
Really solid solution (5 stars)

### DISCUSSION THREAD

**Discussion by Milswanca** [AndreaCatania/RuntimeSkeletalMeshGenerator#6] [General]
Just saying this is an extremely well written plugin and does the job perfectly. You saved me a lot of pain.

![image](https://user-images.githubusercontent.com/11529509/157373791-cc317d1e-0c4e-4f9a-ab04-133e16345c04.png)

This static_cast trick is the (most perfect) sneakiest little trick I've ever seen. Nice work!!

**Comment by AndreaCatania** [comment]
Hehe, thanks a lot :)
